readme.md

```diff
@@ -0,0 +1,43 @@
+# tesr.finance 
+
+## Initial setup
+
+- Fork the original [repository](https://test.com) in to your github account
```

Comment:
I guess this should be https://github.com/MVPWorkshop/tesseract-app instead. Regarding the comment above, we can remove this step and use this link in `git clone`